### DISCUSSION TITLE
Documentation for A2W Dataservices

### DISCUSSION THREAD

**Discussion by andyvan-trabus** [USACE/cwms-data-api#287] [Q&A]
I'm a contractor trying to get data from https://water.usace.army.mil/a2w/f?p=100:1::::::#.  The "Access the A2W Dataservices API" link leads to error 404, and the A2W Help link for "Is there an API to access A2W data" leads me to this repo.

I tried the Example Request on this page (https://github.com/USACE/cwms-radar-api/wiki/TimeSeries), but I got an error 404 that "cwms-public.usace.army.mil took too long to respond."  So, looks like the examples aren't up-to-date.

I did find this link: https://cwms-data.usace.army.mil/cwms-data/swagger-ui.html, but I'm still trying to figure how which API to use to get what I need.

How do I request out_flow time-series for a dam?

**Comment by MikeNeilson** [comment]
Thank you for pointing that out. A bug in the code that processes the desired format was introduced. I'll be fixing that this morning and will try to provide some additional examples.

If there is a specific dam, or set of dam, you know the outflow for please let us know. I'll use it for the additional examples.

**Reply by andyvan-trabus** [reply]
The Joe Pool Dam/Lake in Texas would be good.

**Reply by MikeNeilson** [reply]
Okay, should be working again and I've created another guide on the wiki for that specific data: https://github.com/USACE/cwms-radar-api/wiki/Finding-Data

**Reply by andyvan-trabus** [reply]
Thank you for writing that.  It makes sense.  Unfortunately the server seems to be down at the moment, so I'm unable to test this out.

**Reply by MikeNeilson** [reply]
it's up, but it does look like I screwed up some of the links.

**Reply by MikeNeilson** [reply]
Try it again. The links weren't formatting right a few had the wrong host name.
